### PR TITLE
Lxc 176

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

**Comment by rotty3000** [comment]
tested in windows

File: cmd/ext/create.go [hunk 1]
```diff
@@ -226,9 +227,14 @@ func getWorkspaceProjects() []string {
 	e := filepath.Walk(
 		workspaceDir,
 		func(path string, info os.FileInfo, err error) error {
-			if err == nil && info.Name() == "client-extension.yaml" {
-				fullPathDir := filepath.Dir(path)
-				projectSet[fullPathDir[len(workspaceDir):]] = novalue
+			if err == nil {
+				if strings.HasPrefix(path, filepath.Join(workspaceDir, ".git")) {
```
Comment (gamerson) [inline cmd/ext/create.go:231]: Maybe we should ignore all .gitignore folders too?  So we can avoid things like node_modules?

File: cmd/sync/sync.go [hunk 2]
```diff
@@ -12,9 +12,10 @@ import (
 
 // syncCmd represents the config command
 var syncCmd = &cobra.Command{
-	Use:   "sync [OPTIONS] [ARG...]",
-	Short: "Sync the backing git repo of liferay cli",
-	Args:  cobra.NoArgs,
+	Use:     "sync [OPTIONS] [ARG...]",
+	Short:   "Sync the backing git repo of liferay cli",
+	Aliases: []string{"s"},
```
Comment (gamerson) [inline cmd/sync/sync.go:17]: Will this get us into trouble in the future? If we have another command that might want to be shortended to `s`?

File: docker/docker.go [hunk 3]
```diff
@@ -26,14 +26,14 @@ import (
...
 	"github.com/docker/docker/api/types/container"
-	"github.com/docker/docker/builder/dockerignore"
 	"github.com/docker/docker/client"
 	"github.com/docker/docker/pkg/archive"
 	"github.com/docker/docker/pkg/fileutils"
 	"github.com/docker/docker/pkg/idtools"
 	"github.com/docker/docker/pkg/jsonmessage"
 	"github.com/docker/docker/pkg/progress"
 	"github.com/docker/docker/pkg/streamformatter"
+	"github.com/moby/buildkit/frontend/dockerfile/dockerignore"
```
Comment (gamerson) [inline docker/docker.go:36]: Is this on purpose?

File: cmd/ext/create.go (hunk 1 above)
Comment (rotty3000) [reply cmd/ext/create.go:231]: hmm, perhaps that would be more performant (not that it took more than a few millis but sure)

File: docker/docker.go (hunk 3 above)
Comment (rotty3000) [reply docker/docker.go:36]: yes, resolving a deprecation warning.

File: cmd/sync/sync.go (hunk 2 above)
Comment (rotty3000) [reply cmd/sync/sync.go:17]: perhaps. it may have been a little frivolous so i can remove it.

Comment (gamerson) [reply cmd/sync/sync.go:17]: its a 0.x so we can always change aliases in the future.  we can leave it i think.

File: cmd/ext/create.go (hunk 1 above)
Comment (rotty3000) [reply cmd/ext/create.go:231]: I fear building a .gitignore parser might not be worth the effort here. I'll look to see if there's a go library which does this.

Comment (gamerson) [reply cmd/ext/create.go:231]: Yes, I was thinking in my head if there was a library that could walk files and skip any .{foo}ignores.

Comment (rotty3000) [reply cmd/ext/create.go:231]: skipping this as discussed

**Pull request merged**